plugin/src/main/java/org/opensearch/ml/rest/RestMLStatsAction.java

```diff
@@ -176,7 +176,11 @@ public void onResponse(SearchResponse searchResponse) {
 
                     @Override
                     public void onFailure(Exception e) {
-                        onFailed(channel, RestStatus.INTERNAL_SERVER_ERROR, "Searching model wasn't successful", e);
+                        try {
+                            getNodeStats(finalMlStatsInput, clusterStatsMap, client, mlStatsNodesRequest, channel);
```

Comment:
Shouldn't we check if the failure is due to index not found exception and continue the stats execution? For every other failures, we can throw exception.

Reply:
okay makes sense. user should know nothing of hidden model. got it